stories/setup/config.tsx

```diff
@@ -22,7 +22,7 @@ import './styles.scss';
 configureApplication();
 
 // Automatically import all files ending in *.stories.tsx
-const req = require.context('../', true, /.stories.tsx$/);
+const req = require.context('../', true, / '../',/stories.tsx$/);
```

Comment:
This isn't valid syntax. What were you trying to do exactly?

<img width="800" alt="Screenshot 2019-11-04 11 29 51" src="https://user-images.githubusercontent.com/55398/68143166-7fbccb80-fef6-11e9-9ff0-55be48a64620.png">

Reply:
Oh, maybe it was meant to be this?

```tsx
const req = require.context('../', true, '../', /stories.tsx$/);
```